lib/bridge.js

```diff
@@ -534,6 +572,25 @@ Bridge.prototype.getBot = function() {
     return this._appServiceBot;
 };
 
+/**
+ * Can the room be provisioned based on the rules provided to the
+ * bridge.
```

Comment:
questions end with question marks. Alternatively, it could be rephrased as "Whether or not the room can be..."